.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+Thank you for contributing to prometheus-community/helm-charts. Before you submit this PR we'd like to make sure you are aware of our technical requirements and best practices:
+
+* https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#technical-requirements
+* https://helm.sh/docs/chart_best_practices/
+
+For a quick overview across what we will look at reviewing your PR, please read our review guidelines:
+
+// TODO: add a REVIEW_GUIDELINES.md in prometheus-community/helm-charts
```

Comment:
@scottrigby @torstenwalter I have added a TODO comment here to add the review guidelines for prometheus-community/helm-charts. For now, this will refer the helm/charts' guidelines in the PR